pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -42,6 +42,7 @@ import (
 
 const (
 	launchTemplateNameFormat  = "Karpenter-%s-%s"
+	karpenterManagedTagKey    = "karpenter.k8s.aws/managed-by"
```

Comment:
Thoughts on making it more clear this is cluster name?
```suggestion
	karpenterManagedTagKey    = "karpenter.k8s.aws/cluster"
```